unidist/core/backends/mpi/core/serialization.py

```diff
@@ -216,3 +216,67 @@ def deserialize(self, s_data):
         unpacked_data = msgpack.unpackb(s_data, object_hook=self._decode_custom)
         gc.enable()
         return unpacked_data
+
+
+class SimpleSerializer:
+    """
+    Class for simple data serialization/de-serialization for MPI comminication.
+
+    Notes
+    -----
+    Uses cloudpickle and pickle libraries as a separate APIs
```

Comment:
What's need of `pickle` using? Why is `cloudpickle` not enough?

Reply:
dot at the end of this sentence.